### DISCUSSION TITLE
Nowy filtr do blokowania irytujących rzeczy stworzony przeze mnie.

### DISCUSSION THREAD

**Discussion by seba2282** [MajkiIT/polish-ads-filter#18746] [General]
Witam, jak już wiecie, stworzyłem nowy filtr do blokowania irytujących rzeczy na stronach. I na razie się uczę, więc to jest testowo. Informacje znajdziecie tutaj:
https://gist.github.com/seba2282/7b517bcf0b6f75a38be8f51f69539111
Nie wiem czy na razie dobrze wszystko wygląda w kodzie z tymi regułami. Potrzebowałbym też pomocy fachowców, jak @krystian3w, @AdamWr, @MajkiIT i wielu innych. Oczywiście informujcie mnie, jakie reguły usunąć z tego filtra, bo dodaliście w głównych polskich filtrach. Liczę na współpracę (aktualnie w możliwościach, jak będę umiał pomóc; z czasem na normalną w rozwijaniu filtrów). Cieszmy się wolnym internetem bez śmieci reklamowych i innych dodatków irytujących. :D

**Comment by xxcriticxx** [comment]
Congratulations!

**Comment by seba2282** [comment]
Dzięki :)

**Comment by hawkeye116477** [comment]
Do irytujących rzeczych już przecież jest https://github.com/PolishFiltersTeam/PolishAnnoyanceFilters/ i https://github.com/PolishFiltersTeam/PolishAntiAnnoyingSpecialSupplement :stuck_out_tongue:

**Reply by seba2282** [reply]
@hawkeye116477 To dlaczego są ci partnerzy? Albo zaloguj za pomocą... Na stronach, itp.? 😛

**Reply by hawkeye116477** [reply]
Po prostu nasze filtry nie są aż tak restrykcyjne, raczej chowamy głównie wielkie, ruchome i oczywiste. Logowania nie usuwamy, bo jest przydatne (nawet sam korzystam z tego na Epicu) :smile:.

Tak więc rozumiem, że twoja lista ma być raczej kontynuacją projektu Konoromi.

**Reply by seba2282** [reply]
@hawkeye116477 Ale "partnerzy" moglibyście usuwać. 😅

**Reply by krystian3w** [reply]
MajkIT chował jako "reklama", pewnie stąd żaden inny projekt nie chciał chować w momencie jak `@xMaap` takie wykrywał i kończyło się to zamknięciem bez ukrycia.

Niby na dobrychprogramach nie ma już listy partnerów w okolicy stopki a ja jedynie kojarzę coś o stronie "GSM" gdzie niby po raz ostatni takie coś ukryto jako "reklama".

> https://github.com/MajkiIT/polish-ads-filter/blob/a1b876654a1eaa91c491d2512f8227a3acf1cd72/polish-adblock-filters/adblock.txt#L2164 - chyba to jest ten element, niby strona też już nie ma patronów/partnerów medialnych-reklamowych.

---

Pewnie temat dyskusyjny, jak blokada ankiet w kolumnach bocznych, bo Blocker ich nie lubi ale zazwyczaj jakby ich nie widział, mimo że na zgłaszanych stronach są.

**Reply by seba2282** [reply]
> Pewnie temat dyskusyjny, jak blokada ankiet w kolumnach bocznych, bo Blocker ich nie lubi ale zazwyczaj jakby ich nie widział, mimo że na zgłaszanych stronach są.


Bardzo możliwe, @krystian3w

**Reply by seba2282** [reply]
> Po prostu nasze filtry nie są aż tak restrykcyjne, raczej chowamy głównie wielkie, ruchome i oczywiste. Logowania nie usuwamy, bo jest przydatne (nawet sam korzystam z tego na Epicu) :smile:.
> 
> Tak więc rozumiem, że twoja lista ma być raczej kontynuacją projektu Konoromi.


Ostatnio jest problematyczny ten web annoyances Ultralist. I widzę niewiele reguł na strony PL. Te 3 co zgłosiłem dla centrum xp są. Aczkolwiek jest dodatkowy element, który jest odnośnikiem do tego sklepu i potrzeba go dodać także. Cały czas się zastanawiam czy nie porzucę używanie web annoyances Ultralist w dodanych filtrach blokowania.

**Comment by seba2282** [comment]
Używacie może Web Annoyances Ultralist? Czy raczej rezygnujecie z tego filtru? Jeżeli mogę spytać. I co się stało, Easylist Polish znikł z filterlists.com?

**Reply by krystian3w** [reply]
Stamtąd nie zniknął, Krzysiek z Norwegii by się nie zgodził: https://filterlists.com/lists/easylist-polish

https://github.com/collinbarrett/FilterLists/pull/374 / https://github.com/collinbarrett/FilterLists/blob/0b7b261296f3c8d8e87f051976fb62278340a3f0/services/Directory/data/FilterList.json#L3126-L3133 (plik sprzed 2 tygodni)

**Reply by seba2282** [reply]
Dziwne, nie mogłem go znaleźć ostatnio przez microsoft edge i myślałem, że się zwinął z filterlists. 


A używasz Web Annoyances Ultralist Filter? Bo nie czy nie zrezygnować z niego i na ile zamierzam widzieć znacznie więcej reklam bez tego filtra.
